src/Compilers/CSharp/Portable/Binder/ExpressionVariableFinder.cs

```diff
@@ -390,6 +393,16 @@ public override void VisitConstructorDeclaration(ConstructorDeclarationSyntax no
             }
         }
 
+        public override void VisitRecordDeclaration(RecordDeclarationSyntax node)
+        {
+            Debug.Assert(((RecordDeclarationSyntax)node).ParameterList is object);
```

Comment:
>RecordDeclarationSyntax [](start = 27, length = 23)

nit: redundant cast

Reply:
> nit: redundant cast

Removed in [https://github.com/dotnet/roslyn/pull/44906](https://github.com/dotnet/roslyn/pull/44906)

---
In reply to: [436172247](https://github.com/dotnet/roslyn/pull/44876#discussion_r436172247) [](ancestors = 436172247)